lib/matplotlib/patches.py

```diff
@@ -707,7 +707,7 @@ def __init__(self, xy, width, height, angle=0.0, **kwargs):
         Parameters
         ----------
         xy : (float, float)
-            The bottom and left rectangle coordinates
+            The starting coordinates. For most data plotting functions, it is (left, bottom). For imshow(), it is (left, top).
```

Comment:
This has to fit on a line <=79 characters long.  I'd just say "minimum x and y coordinate of rectangle corner"

Reply:
Sure, it will be clear to use "minimum x and y coordinate of rectangle corner". Thank you.

Reply:
Minimum is only true if the the angle is zero and the width/height is positive.  
So "starting coordinates" would be fine, or maybe "anchor point" or so.